src/NzbDrone.Core/MediaFiles/EpisodeImport/Aggregation/Aggregators/Augmenters/Quality/AugmentQualityFromFolder.cs

```diff
@@ -13,10 +15,18 @@ public AugmentQualityResult AugmentQuality(LocalEpisode localEpisode)
                 return null;
             }
 
+            var sourceConfidence = quality.SourceDetectionSource == QualityDetectionSource.Unknown
```

Comment:
Maybe checking against `Name` explicitly is better (returning Tag). In case the folder name contains an extension (returning Fallback).